ci/stack-orchestration/configuration/dev/vpc/parameters.json

```diff
@@ -30,5 +30,9 @@
   {
     "ParameterKey": "DynamoDBApiEnabled",
     "ParameterValue": "Yes"
+  },
+  {
+    "ParameterKey": "LambdaApiEnabled",
+    "ParameterValue": "Yes"
```

Comment:
I've only edited the parameters for dev, not other envs for now, as making changes to the VPC will require careful testing in dev. 
ATO-694 (writing the code to invoke FetchJwksHandler) should be completed first in order to do proper testing.